modules/programs/k9s.nix

```
@@ -0,0 +1,70 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.k9s;
+  yamlFormat = pkgs.formats.yaml { };
+in {
+  meta.maintainers = [ hm.maintainers.katexochen ];
+
+  options.programs.k9s = {
+    enable =
+      mkEnableOption "k9s - Kubernetes CLI To Manage Your Clusters In Style!";
```

`mkEnableOption` will add a `.` in the end of the string so this would yield `!.`

```suggestion
      mkEnableOption "k9s - Kubernetes CLI To Manage Your Clusters In Style";
```